src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state: Readonly<{ key?: string }> = {
```

Comment:
Now key always is string 
```suggestion
  state: Readonly<{ key: string }> = {
```

Reply:
and create separate interface for it